packages/sui-lint/Readme.md

```diff
@@ -52,6 +52,22 @@ Lints all `**/src/**/*.scss` files in the project, excluding `node_modules`, `li
 
 > **`.gitignore` file patterns are also excluded but interpretation may differ as only glob patterns are understood**
 
+
+### Scope commands to staged files
+
+```sh
+$ sui-lint js --staged
+$ sui-lint sass --staged
+$ sui-lint format-js --staged
+```
+
+Same command but applied only on staged files (obtained with `git diff --cached --name-only --diff-filter=d` command).
+
+> **In write mode like with `sui-lint js --fix` or `sui-lint format-js`, the `--staged` option will stage the files again (`git add <file...>`)**
```

Comment:
We could also to a new option `--add` or `--commit` that will do only this and not make it auto.

Reply:
I think that can be dangerous add files without my control. For example I usually add file to the stage by chunks. And this feature can add thing to the commit that I really dont want add yet

Maybe with a warning message could be enough

Reply:
You're right. I'll add a "--add-fixes" flag to do that. Is it ok?